nav2_core/include/nav2_core/local_planner_base.hpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Software License Agreement (BSD License)
```

Comment:
Is this really a new file? If so, why the copyright Locus robotics and the BSD license?

Reply:
He copy pasted the robot_navigation local planner interface and added the lifecycle support.

Reply:
@mkhansen-intel Yes, on what Steve said above. What is the best way to go in this scenario?